tsl/test/sql/chunk_utils_internal.sql

```diff
@@ -517,10 +515,6 @@ SELECT _timescaledb_internal.stop_background_workers();
 
```

Comment:
The tests in Lines 497 - 516 (`-- Test dump & restore`) can also be removed. They were introduced to test backup and restore when the block trigger is active. In addition, Lines 133 - 140 in `tsl/test/sql/CMakeLists.txt`can be removed.

Reply:
done